test/fixtures/track-long-names.json

```diff
@@ -40,6 +40,7 @@
       {
         "_eventName": "Testing in case someone sends us events ",
```

Comment:
ah, sorry, I was referring to **bug 1**. This test case appears to me to be an unmapped event name that is mapped correctly (ie. not "null")